go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -108,10 +118,11 @@ func (agent *ActionAgent) restoreDataLocked(ctx context.Context, logger logutil.
 	case nil:
 		// Starting from here we won't be able to recover if we get stopped by a cancelled
 		// context. Thus we use the background context to get through to the finish.
-
-		// Reconnect to master.
-		if err := agent.startReplication(context.Background(), pos, originalType); err != nil {
-			return err
+		if keyspace.KeyspaceType == topodatapb.KeyspaceType_NORMAL {
+			// Reconnect to master only for "NORMAL" keyspaces
```

Comment:
Do we also need to skip replication repair in the replication reporter?

Reply:
It seems cleaner to require people to run the tablets with `-enable_replication_reporter=false` than to clutter the replication reporter.

Reply:
I have added code to force-set this to false while restoring snapshots.